src/source/Ice/IceAgent.c

```diff
@@ -1084,12 +1083,12 @@ STATUS createIceCandidatePairs(PIceAgent pIceAgent, PIceCandidate pIceCandidate,
     PIceCandidatePair pIceCandidatePair = NULL;
     BOOL freeObjOnFailure = TRUE;
     PIceCandidate pCurrentIceCandidate = NULL;
-    BOOL statsControlEnabled = FALSE;
+    BOOL disableStats = FALSE;
```

Comment:
I would suggest negating this to `enableStats`. Would make it easier to read and follow (avoiding double-negatives)

Reply:
I initially was going to do just that and name it `enableStats`, but it seemed having both an `enableStats` local BOOL and an `enableIceStats` member - one of which is FALSE by default and the other not even compiled by default - might be confusing. I will rename it to a negated `doStatCalcs`.